tests/relay/test_exceptions.py

```diff
@@ -26,6 +26,16 @@ class Fruit(relay.Node):
         code: str
 
 
+def test_not_raises_error_on_missing_node_id_annotation_but_resolve_id():
+    @strawberry.type
+    class Fruit(relay.Node):
+        code: str
+
+        @classmethod
+        def resolve_id(cls, root) -> str:
+            return "test"
```

Comment:
Some things here:

1) This doesn't raise an exception, so probably the best place to put this is on relay tests itself. Maybe on `relay/test_types.py`?

2) Just to make it more complete, it would be nice to create a schema here and compare the it. Search for `dedent` on the code and you will find some tests that do that as an example

Reply:
thanks for pointing to the right file